src/modules/apps/e2e/report/failedApps.tsx

```diff
@@ -80,17 +90,28 @@ const ErrorVisualization: React.FunctionComponent<ErrorVisualizationProps> = ({
   error,
   stack,
   testScreenshots,
+  testVideo,
+  specId,
+  testId,
 }) => {
-  const testScreenshotsText = testScreenshots.map(curScreenshot => ` ${curScreenshot.path}`).join('\n')
+  const { account, workspace } = SessionManager.getSingleton()
+  const testScreenshotsUrl = testScreenshots
+    .map(
+      curScreenshot =>
```

Comment:
Could you create a function:
```
const workspaceBaseURL = () => {
  const { account, workspace } = SessionManager.getSingleton()
  return `${workspace}--${account}.${publicEndpoint()}`
}
```
Ideally this should be a common function, but for now there isn't one like that :(

(the `publicEndpoint` function is from `env.ts`)